drivers/sensor/paw3212/paw3212.c

```diff
@@ -634,6 +767,7 @@ static int paw3212_attr_set(struct device *dev, enum sensor_channel chan,
 			    const struct sensor_value *val)
 {
 	struct paw3212_data *dev_data = &paw3212_data;
+	int err;
```

Comment:
I guess the function should return this (currently it returns 0 when the actual attribute change fails).

Reply:
Thanks! fixed